### ISSUE TITLE
Duplicate instantiation of AppKernel in app.php

### ISSUE THREAD

**Issue by cristi-contiu** [symfony/symfony-demo#47]
Hello.

According to the comments in app.php, in order to enable the Symfony Reverse Proxy, you must uncomment the lines:

```
// require_once __DIR__.'/../app/AppCache.php';
// $kernel = new AppKernel('prod', false);
// $kernel->loadClassCache();
// $kernel = new AppCache($kernel);
// Request::enableHttpMethodParameterOverride();
```

However, the AppKernel is instantied and the ClassCache is loaded right before this block. 

Is there a reason for re-constructing the kernel and loading the ClassCache only after
`require_once __DIR__.'/../app/AppCache.php';`?

Thank you!

**Comment by stof** [comment]
This indeed looks like an issue

**Comment by javiereguiluz** [comment]
@cristi-contiu thanks for reporting this issue.

The purpose was to provide an easy way to comment a block of code and uncomment another block to easily enable the HTTP cache. As you know, the current front controller of the standard edition mangles these two code blocks by commenting some lines of code.

This practice looks confusing to me and I have replied to lots of user support questions about this. That's whay I wanted to provide two similar but totally separated code blocks. In any case, you are right and the original controller had some issues. I've updated its code in #50.
